src/common/createSlider.jsx

```diff
@@ -120,6 +120,22 @@ export default function createSlider(Component) {
       utils.pauseEvent(e);
     }
 
+    onFocus = (e) => {
+      const isVertical = this.props.vertical;
+
+      if (utils.isEventFromHandle(e, this.handlesRefs)) {
+        const handlePosition = utils.getHandleCenterPosition(isVertical, e.target);
+
+        this.dragOffset = 0;
```

Comment:
I think I did this to "reset" mouse/touch behaviours when combining keyboard with these. But I'm not sure if this is even necessary. Just tested with and without this and I had the same outcome... I can remove it if you want?